workspaces/jenkins/plugins/jenkins/src/alpha/entityCards.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { createSchemaFromZod } from '@backstage/frontend-plugin-api';
+import { createEntityCardExtension } from '@backstage/plugin-catalog-react/alpha';
+
+/**
+ * @alpha
+ */
+export const entityLatestJenkinsRunCard = createEntityCardExtension({
+  name: 'latest-run',
+  configSchema: createSchemaFromZod(z =>
```

Comment:
can you add a default filter here (and within entityContent) for the right entity kinds that these cards/content should appear on? More context [here](https://spotify.slack.com/archives/C04DD1C1PEF/p1724359881581689), example PR [here](https://github.com/backstage/community-plugins/pull/905/files#diff-9ca4336ff0762c8d670c7d4d5cfe33557ff4cbe3f6d2560fb0dbd0b5b52b0bd8)